packages/account/src/Sections/Profile/PhoneVerification/__test__/cancel-phone-verification-modal.spec.tsx

```diff
@@ -25,15 +26,27 @@ describe('CancelPhoneVerificationModal', () => {
         document.body.removeChild(modal_root_el);
     });
 
+    const mock_store = mockStore({
+        ui: {
+            is_mobile: false,
```

Comment:
Is this needed? Default config of `mockStore` has the ui value as false

Reply:
Will remove it 🙏